persistence-modules/spring-data-jpa-4/src/main/resources/application.properties

```diff
@@ -1,5 +1,10 @@
 spring.jpa.show-sql=true
-#MySql
-#spring.datasource.url=jdbc:mysql://localhost:3306/baeldung
-#spring.datasource.username=baeldung
-#spring.datasource.password=baeldung
\ No newline at end of file
+spring.datasource.url=jdbc:h2:mem:baeldung
+
+# JPA-Schema-Generation
+# Use bellow configuration to generate database schema create commands based on the entity models
```

Comment:
`below` - `bellow` means to SHOUT LOUD! :)

Reply:
Thanks for the hint :) Done